applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_process_utilities.cpp

```diff
@@ -218,26 +212,19 @@ KRATOS_TEST_CASE_IN_SUITE(AddProcessesSubModelPartList_AddsFilledList, KratosGeo
 
     auto solver_settings = project_parameters["solver_settings"];
 
-    std::stringstream buffer;
-    std::streambuf*   old = std::cout.rdbuf(buffer.rdbuf()); // NOSONAR
-
     // Act
-    ProcessUtilities::AddProcessesSubModelPartList(project_parameters, solver_settings);
+    ProcessUtilities::AddProcessesSubModelPartListToSolverSettings(project_parameters, solver_settings);
 
     // Assert
-    std::cout.rdbuf(old); // NOSONAR
-    std::string output = buffer.str();
-    EXPECT_NE(output.find("processes_sub_model_part_list is deprecated"), std::string::npos);
-
     const auto& r_list = solver_settings["processes_sub_model_part_list"];
     KRATOS_EXPECT_EQ(r_list.size(), 4);
 
-    std::set<std::string, std::less<>> actual_modelpart_names;
-    for (const auto& r_name : r_list) {
-        actual_modelpart_names.insert(r_name.GetString());
-    }
-    const std::set<std::string, std::less<>> expected_modelpart_names = {"Sides", "BottomFixed",
-                                                                         "TopLoad", "Soil"};
+    std::vector<std::string> actual_modelpart_names;
+    std::transform(r_list.begin(), r_list.end(),
+                   std::inserter(actual_modelpart_names, actual_modelpart_names.end()),
+                   [](const auto& r_name) { return r_name.GetString(); });
```

Comment:
Am I right when I say that the following code gives you the same result:
```suggestion
    const auto actual_modelpart_names = solver_settings["processes_sub_model_part_list"].GetStringArray();
```
In that case, we no longer need `r_list`.

Reply:
Wow, awesome!